.github/workflows/labeler.yml

```
@@ -32,3 +32,39 @@ jobs:
     - name: "Extract commit scope and add as label"
       continue-on-error: true
       run: gh pr edit "$PR_NUMBER" --add-label "$(echo "$PR_TITLE" | sed -E 's|[[:alpha:]]+\((.+)\)!?:.*|\1|')"
+
+  add-reviewer:
+    runs-on: ubuntu-latest
+    needs: ["triage", "type-scope"]
+    steps:
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            const reviewers = []
+
+            const { data: { labels: labels } } = await github.rest.pulls.get({
```

This should work too:
```suggestion
            const { data: { labels } } = await github.rest.pulls.get({
```